src/RevPrefetcher.cc

```diff
@@ -156,6 +156,7 @@ void RevPrefetcher::Fill(uint64_t Addr){
   if((Addr%4)!=0){
     // not 32bit aligned, adjust the base address by 2 bytes
     Fill(Addr&0xFFFFFFFFFFFFFFFC);
+    return;
   }
```

Comment:
w/o this exit we end up filling twice... once on the aligned address, and a 2nd time on the unaligned address